### DISCUSSION TITLE
How to configure a new stream

### DISCUSSION THREAD

**Discussion by ralphshep** [NginxProxyManager/nginx-proxy-manager#2106] [Q&A]
Hello,
I'm trying to pass my Minecraft server and OpenVPN server from my local network to my domain (ex: example.com) to be accessed remotely and securely through SSL and/or proxies. 

Clicking the (?) tell you to want you can do with steam, like run a game server, FTP, or SSH, and that it forwards TCP/UDP traffic. But, I can't seem to find any documentation on how to configure a new stream. 

Does anyone have any resources?

![Screen Shot 2022-06-12 at 2 39 43 PM](https://user-images.githubusercontent.com/52576888/173248298-81004f7d-68c6-4d09-8c73-e18f82d07259.png)

**Comment by ammarjmhor10** [comment]
hi [ralphshep](https://github.com/ralphshep), thanks for your question this is the same problem I faced. the solution is simple check the incoming port if it open or not

**Reply by OldestBen** [reply]
> hi [ralphshep](https://github.com/ralphshep), thanks for your question this is the same problem I faced. the solution is simple check the incoming port if it open or not

Is this the incoming port of the container or the incoming WAN port?

**Comment by techsk8** [comment]
AFAICS, until this point nobody on the internet seems to properly explain how to setup a stream for a service like SSH or FTP with npm.
